hq/app/views/fragments/openSecurityGroups.scala.html

```diff
@@ -8,14 +8,16 @@
 @resourceList(flaggedSg: SGOpenPortsDetail, usages: Set[SGInUse]) = {
     @usages.map {
         case Instance(id) => {
-              <a class="btn usage-cta" href="https://@{flaggedSg.region}.console.aws.amazon.com/ec2/v2/home?region=@{flaggedSg.region}#Instances:search=@id">
+              <a target="_blank" rel="noopener noreferrer" class="btn usage-cta" href="https://@{flaggedSg.region}.console.aws.amazon.com/ec2/v2/home?region=@{flaggedSg.region}#Instances:search=@id">
```

Comment:
I'm not sure we need blank targets, our users know how to use web browsers (and can choose to open in a new tab if they want to)?

Reply:
when me & Kate visited teams. we noticed most people didn't do `ctrl/cmd + click` and even asked for opening in a new tab.

Reply:
Yep - several people requested new tab default after clicking through to AWS, then realising that they had lost the tab for Security HQ.

Not a huge fan of target blank, but seems here it solves a fairly common UX issue. However, I would like to keep a watch on further usage!

Reply:
:eyes: :+1: